### ISSUE TITLE
For a full El Capitan feel the search selection color could be a darker blue

### ISSUE THREAD

**Issue by benjaminwil** [benjaminwil/LaunchBar-ElCapitanSmall#3]

**Comment by benjaminwil** [comment]
![au5hpr9epcodcuqr0tefqfzmjvb0ehmzfzhwvkwxok0](https://cloud.githubusercontent.com/assets/883581/11787007/86e5d64a-a23e-11e5-95bb-3922a9d553f5.png)

**Comment by vancura** [comment]
FYI there is no _standard_ in selection background colors, it really depends on the developer and even Apple has no unified color.

**Comment by vancura** [comment]
But if you want to make it native-ish looking, remove that text shadow :)

**Comment by benjaminwil** [comment]
Thanks for the comments @vancura!

Gonna look into fixing the text shadow, thanks. Let me know if there's anything else you'd change.

As far as a "standard" selection colour, I think you're right. I was trying to match the gradient Apple uses "For Buttons, Menus, and Windows" in **System Preferences > General**. Probably good enough? 

![screen shot 2015-12-14 at 10 36 39 am](https://cloud.githubusercontent.com/assets/883581/11790171/2ecaaf4c-a24f-11e5-80a0-96b562ec6d7a.png)

This issue was brought up originally in [the LaunchBar forum](https://forums.obdev.at/viewtopic.php?f=4&t=10115) and I haven't totally checked it out yet. Don't know exactly what I'm up to yet.

**Comment by vancura** [comment]
It's looking great! That's exactly the most used color.

BTW here's my approach, which I made nearly two years ago, when the last LB major version was launched, and Yosemite was everywhere. Since then I just finetuned fonts so it now uses San Francisco. Otherwise it's so unpolished and unfinished I never really wanted to release it :)

<img width="1080" alt="tmp" src="https://cloud.githubusercontent.com/assets/4982/11790470/50279b30-a29c-11e5-99e4-c971feee975b.png">
